tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "suppressExcessPropertyErrors": true,
-    "strictNullChecks": false
+    "strictNullChecks": false,
+    "noImplicitAny": false
```

Comment:
I'm not in favour of this, open discussion on the channel. The power of TS is the enforcing of types, if you do this, you allow const something; instead of const somethig: Type

Reply:
i agree - @kevinhcolgan what is the cost of fixing this code to avoid needing this?